deliver/lib/deliver/app_screenshot.rb

```diff
@@ -293,13 +302,27 @@ def self.devices
       }
     end
 
+    def self.resolve_ipadpro_conflict_if_needed(device_type, file_name)
+      is_3rd_gen = file_name.include?("(3rd generation)")
```

Comment:
This behavior is tricky!

Could you update documentation about this?
https://github.com/fastlane/docs/blob/master/docs/actions/upload_to_app_store.md